pkg/quarterdeck/auth_test.go

```diff
@@ -116,6 +117,7 @@ func (s *quarterdeckTestSuite) TestRegister() {
 	s.CheckError(err, http.StatusConflict, "user or organization already exists")
 
 	// Test that one verify email was sent to each user
+	s.StopTasks()
```

Comment:
I think this should avoid the race condition for now, let me know if there's a more delicate way to wait for the current tasks to stop without having to reset the entire task manager.

Reply:
Ah, I didn't think about race conditions with respect to the tests. You're right that this is a pretty tricky situation.